handlers/twiml/handlers.go

```diff
@@ -235,36 +236,31 @@ func (h *handler) Send(ctx context.Context, msg courier.MsgOut, res *courier.Sen
 		return err
 	}
 
-	parts := handlers.SplitMsgByChannel(msg.Channel(), msg.Text(), maxMsgLength)
-	for i, part := range parts {
-		// build our request
+	// do we have a template and support whatsapp scheme?
+	if msg.Templating() != nil && channel.IsScheme(urns.WhatsApp) {
+		if msg.Templating().ExternalID == "" {
+			return courier.ErrMessageInvalid
```

Comment:
@norkans7 please have a think about places in other handlers where we should use this

Reply:
Yes